modules/uploading-model-files-to-pvc.adoc

```diff
@@ -0,0 +1,46 @@
+:_module-type: PROCEDURE
+
+[id="uploading-model-files-to-pvc_{context}"]
+
+= Uploading model files to a Persistent Volume Claim (PVC)
+
+When deploying a model, you can serve it from a preexisting Persistent Volume Claim (PVC) where your model files are stored. You can upload your local model files to a PVC by using a running workbench.
+
+.Prerequisites
+
+* You have access to the {productname-short} dashboard.
+* You have a running workbench.
+ifndef::upstream[]
+* The workbench is attached to a persistent volume (PVC), as described in link:{rhoaidocshome}{default-format-url}/working_on_data_science_projects/using-project-workbenches_projects#creating-a-project-workbench_projects[Creating a project workbench].
+endif::[]
+ifdef::upstream[]
+* The workbench is attached to a persistent volume (PVC), as described in link:{odhdocshome}/working-on-data-science-projects/#creating-a-workbench-select-ide_projects[Creating a project workbench].
+endif::[]
+* You have the model files saved on your local machine.
+
+.Procedure
+
+Follow these steps to upload your model files to the PVC:
+
+. From the {productname-short} dashboard, launch your running workbench.
```

Comment:
suggest:
.  From the {productname-short} dashboard, click the open icon (image:images/open.png[The open icon]) to open your IDE in a new window.
or
.  From the {productname-short} dashboard, click the open icon (image:images/open.png[The open icon]) to open JupyterLab in a new window.